src/extension/_locales/fr/messages.json

```diff
@@ -164,7 +164,22 @@
   "advanced": {
     "message": "Avancées"
   },
+  "config_development": {
+    "message": "Développement"
+  },
   "developer_mode": {
     "message": "Tester le Helix Sidekick localement"
+  },
+  "developer_mode_description": {
+    "message": "Invoke a local development version of the Sidekick"
+  },
+  "admin_version": {
+    "message": "Use a specific Helix Admin API version"
+  },
+  "admin_version_description": {
+    "message": "Caution: This can have unexpected side effects!"
+  },
+  "admin_version_placeholder": {
+    "message": "Admin Version: x.y.z"
```

Comment:
@kptdobe before I have to turn to Google Translation again, could you provide the proper French translations? 😄